code/modules/modular_computers/file_system/programs/engineering/engineering_camera.dm

```diff
@@ -0,0 +1,25 @@
+/datum/computer_file/program/camera_monitor/engineering
```

Comment:
As mentioned by Lohikar it is meant to replace the physical camera consoles around in engineering.
Also if you take a look at the old camera monitor, the access restrictions there are also handled by sub classes.